pkgs/tools/networking/v2raya/default.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, fetchFromGitHub
+, mkYarnPackage
+, buildGo117Module
+, makeWrapper
+, v2ray
+, v2ray-geoip
+, v2ray-domain-list-community
+, symlinkJoin
+}:
+let
+  pname = "v2raya";
+  version = "unstable-2022-02-06";
+  git-rev = "da65d2b4d7c9b2c1646583c02b4e90e4ab15c511";
+  src = fetchFromGitHub {
+    owner = "v2rayA";
+    repo = "v2rayA";
+    rev = git-rev;
+    sha256 = "sha256-Hj7GDuz27OJrMz1JfkyS01dakK5ddV6MxIMh+GrYwNg=";
+  };
+  web = mkYarnPackage {
+    inherit pname version;
+    src = "${src}/gui";
+    yarnNix = ./yarn.nix;
+    packageJSON = ./package.json;
+    yarnLock = ./yarn.lock;
+    buildPhase = ''
+      ln -s $src/postcss.config.js postcss.config.js
+      OUTPUT_DIR=$out yarn --offline build
+    '';
+    distPhase = "true";
+    dontInstall = true;
+    dontFixup = true;
+  };
+in
+buildGo117Module {
```

This is the default.

```suggestion
buildGoModule {
```